Libraries/Lists/VirtualizedList.js

```diff
@@ -1094,8 +1254,7 @@ class VirtualizedList extends React.PureComponent<Props, State> {
         : this.props.style,
     };
 
-    this._hasMore =
-      this.state.last < this.props.getItemCount(this.props.data) - 1;
+    this._hasMore = this.state.viewportWindow.last < itemCount - 1;
```

Comment:
This seems a bit wonky, since it is possible we will never realize up to the end. This ends up blocking parent rendering though. Seems like an existing bug here?